apps/src/gamelab/GameLab.js

```diff
@@ -195,6 +197,12 @@ GameLab.prototype.init = function (config) {
   config.dropletConfig = dropletConfig;
   config.appMsg = msg;
 
+  this.submit = new Submit({
```

Comment:
I feel like we could come up with a more specific name (though not sure exactly what it is). `LevelSubmission`?